docs/src/man/tour.md

```diff
@@ -94,27 +107,30 @@ Hbox = convert(IntervalBox, H)
 typeof(Hbox)
 ```
 
-Hyperrectangles are also special classes of [zonotopes](https://juliareach.github.io/LazySets.jl/dev/lib/sets/Zonotope/).
+Hyperrectangles are a special subclass of
+[zonotopes](https://juliareach.github.io/LazySets.jl/dev/lib/sets/Zonotope/).
 
 ```@example tour
-Z = convert(Zonotope, c)
+Z = convert(Zonotope, C)
 ```
 
-There are many other set representations available in this library. They are introduced
-in further sections. Under doubt, rsemember that by tab-completion is your friend!
+There are many other set representations available in this library. They are
+introduced in further sections below.
```

Comment:
I did not understand what you meant by tab-completion for finding the set representations here.

Reply:
ok. i meant like when you you are in the REPL and type `julia> In[TAB]` to list the possibilities, among them `Interval`.

i'll consider clarifying the statement with an example.